packages/apollo-engine-reporting/src/plugin.ts

```diff
@@ -114,6 +128,37 @@ export const plugin = <TContext>(
         }
       }
 
+      async function shouldTraceOperation(
+        requestContext:
+          | GraphQLRequestContextDidResolveOperation<TContext>
+          | GraphQLRequestContextDidEncounterErrors<TContext>,
+      ): Promise<void> {
+        if (metrics.captureTraces !== undefined)
```

Comment:
In what context will this conditional trigger?

Reply:
I think this would be hit if we call `didEncounterErrors` after `didResolveOperation`. Added a comment.